packages/shared/src/lib/log.ts

```diff
@@ -149,9 +149,19 @@ export enum LogEvent {
   ScheduleStreakReminder = 'schedule streak reminder',
   // 404 page
   View404Page = '404 page',
-  // source
+  // Source Actions - start
+  BlockSource = 'block source',
+  UnblockSource = 'unblock source',
+  // Source Actions - end
   SubscribeSource = 'subscribe source',
   UnsubscribeSource = 'unsubscribe source',
+  // Source Actions - end
+  // Tags - start
+  FollowTag = 'follow tag',
+  UnfollowTag = 'unfollow tag',
+  BlockTag = 'block tag',
+  UnblockTag = 'unblock tag',
+  // Tags - end
```

Comment:
remove those

Reply:
Only if these change from the naming events had - if it's just moving from plain string to enum with the same content then you can leave.